src/main/java/com/example/TicketBookingApp/domain/Ticket.java

```diff
@@ -0,0 +1,33 @@
+package com.example.TicketBookingApp.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder(toBuilder = true)
+public class Ticket {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @OneToOne
+    private Seat seat;
```

Comment:
Tylko jedno miejsce?

Reply:
Tak, moim zdaniem jeden bilet powinien mieć przypisane jedno miejsce. Sugerowałem się własnymi wycieczkami do kina. Zazwyczaj chodzę z żoną i mimo iż bierzemy dwa miejsca na jeden rachunek, to pan przy bramkach zawsze kasuje dwa bilety ;)